README.md

```diff
@@ -11,80 +11,70 @@
 
-## FEC repositories
-We welcome you to explore, make suggestions, and contribute to our code.
-
-This repository, [openFEC](https://github.com/18F/openfec), is home to betaFEC’s API.
-
-### All repositories
+## This repository, [openFEC](https://github.com/18F/openfec), is home to betaFEC’s API
+All FEC repositories:
 - [FEC](https://github.com/18F/fec): a general discussion forum. We [compile feedback](https://github.com/18F/fec/issues) from betaFEC’s feedback widget here, and this is the best place to submit general feedback.
 - [openFEC](https://github.com/18F/openfec): betaFEC’s API
-- [swagger](https://github.com/18F/swagger-ui):this generates our interactive API documentation
-- [openFEC-web-app](https://github.com/18f/openfec-web-app): the betaFEC web app for exploring campaign finance data
+- [swagger-ui](https://github.com/18F/swagger-ui): forked repo that generates our interactive API documentation
+- [openFEC-web-app](https://github.com/18f/openfec-web-app): the web app for exploring campaign finance data
 - [fec-style](https://github.com/18F/fec-style): shared styles and user interface components
-- [fec-cms](https://github.com/18F/fec-cms): the content management system (CMS) for betaFEC
-- [proxy](https://github.com/18F/fec-proxy): this is a light weight app that coordinates the paths between the web app and CMS
+- [fec-cms](https://github.com/18F/fec-cms): this project's content management system (CMS) 
+- [proxy](https://github.com/18F/fec-proxy): this is a lightweight app that coordinates the paths between the web app and CMS
 
 
 ## Get involved
-We’re thrilled you want to get involved!
+We welcome you to explore, make suggestions, and contribute to our code. 
 - Read our [contributing guidelines](https://github.com/18F/openfec/blob/master/CONTRIBUTING.md). Then, [file an issue](https://github.com/18F/fec/issues) or submit a pull request.
-- [Send us an email](mailto:betafeedback@fec.gov).
-- If you’re a developer, follow the installation instructions in the README.md page of each repository to run the apps on your computer.
+- If you'd rather send us an email, [we're thrilled to hear from you](mailto:betafeedback@fec.gov)!
+- Follow our Set up instructions to run the apps on your computer.
 - Check out our StoriesonBoard [FEC story map](https://18f.storiesonboard.com/m/fec) to get a sense of the user needs we'll be addressing in the future.
 
 ---
 
 ## Set up
+We are always trying to improve our documentation. If you have suggestions or run into problems please [file an issue](https://github.com/18F/openFEC/issues)!
 
-Prior to running, ensure you have the following requirements installed:
-<!--requirements-->
-* virtualenv
-* virtualenvwrapper
-* python3.4
-* pip
-* nodejs
-* npm
-* PostgreSQL
 
-<!--endrequirements-->
+### Project prerequisites
+1. Ensure you have the following requirements installed:
 
-## Quick(ish) start
+    * Python 3.4 (which includes pip and virtualenv)
+    * Node.js (which includes npm)
```

Comment:
Given that we list explicit versions for Python and PostGres, we should probably note that for Node.js too - I think including something like `; the latest LTS or stable release is fine` as the last part of what's inside the parentheses is fine.